docs/customizing-evaluation-outputs.md

```diff
@@ -0,0 +1,54 @@
+# Template BPA Customizing Evaluation Outputs
+<a name="note"></a>***Note**: The ARM Template BPA is currently in development. All features that have yet to be implemented have been flagged with an asterisk [\*].*
+
+## Overview
+Template BPA customization is authored in JSON.  Custom evaluation settings are written in a separate configuration and used on tool executing. More details on how to run Template BPA with a configuration file can be found [here](https://github.com/Azure/template-analyzer#using-the-template-bpa). 
+
+## Template BPA Rule Object
+Here are the fields that make up a custom configuration file.
+```javascript
+{
+    "exclusions": { � }, // Parameters that are excluded from the BPA execution.  More details below.
+    "inclusions": { � }, // Only parameters that will be included in the BPA execution.  More details below.
+    "output": { � }, // How the BPA execution output will be displayed to the user.  More details below.
+    "severityReMapping": { � } // Tuple of ruleId and new severity value.  Can change a rule's severity.
+}
+```
+
+### Guidelines for exclusions object
+The `exclusions` is compromised of the following optional properties.
+```javascript
+"exclusions": {
+    "severity": [int], // Means don't include rule with servities that are in this list in the BPA execution
+    "ruleIds": ["ruleId"] // Means don't include rules that are in this list in the BPA execution
+}
+```
+If the inclusions object contains values, then the exclusions object will be ignored. 
+
+### Guidelines for inclusions object
+The `inclusions` is compromised of the following optional properties.
+```javascript
+"inclusions": {
+    "ruleIds": ["ruleId"] // Means only include rules that are in this list in the BPA execution
+}
+```
+If the inclusions object contains values, then the exclusions object will be ignored. 
+
+### Guidelines for output object [[*]](#note)
```

Comment:
Maybe it makes sense to merge this PR after the SARIF one? We will support file output in SARIF format and console output

Reply:
Are we thinking we want to be able to define output both here in the configurations file and on the console, or do we only want on the console?

Reply:
I like the idea of being consistent for all parameters (everything available in here too, or only the options that are not already available as console options) like I tried to say in another comment